### PR TITLE
code defect for null value

### PR DESCRIPTION
**Pull request by kezhenxu94**
image

### CODE REVIEW

**Comment by wu-sheng** [comment]
@kezhenxu94 Haha, you are fixing issues around the whole SkyWalking ecosystem.

**Comment by kezhenxu94** [comment]
> @kezhenxu94 Haha, you are fixing issues around the whole SkyWalking ecosystem.

I'm really interested in SkyWalking and studying the SkyWalking ecosystem recently.

> Can't wait to see RocketBot to become the official UI of SkyWalking and will help to fasten the process of donation if needed :)

**Comment by wu-sheng** [comment]
> P.S. Can't wait to see RocketBot to become the official UI of SkyWalking and will help to fasten the process of donation if needed :)

Welcome to provide this feedback to ml :) Apache like the ml way. I also have submitted a lot of issues in this repo, try to coordinate the donation and make this happens.

**Comment by kezhenxu94** [comment]
> > P.S. Can't wait to see RocketBot to become the official UI of SkyWalking and will help to fasten the process of donation if needed :)
> 
> Welcome to provide this feedback to ml :) Apache like the ml way. I also have submitted a lot of issues in this repo, try to coordinate the donation and make this happens.

I've replied to the ML this morning to vote for the donation and hope it may take effect more or less, I like the Apache way :)

**Comment by kezhenxu94** [comment]
> Sorry you set default value in this way is not good

@TinyAllen any links to explain this? I'm not very familiar with typescript (I'm backend engineer actually)

File: src/store/modules/dashboard-endpoint.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ const mutations: MutationTree<State> = {
   [types.SET_CURRENT_ENDPOINT](state: State, endpoint: any) {
     state.currentEndpoint = endpoint;
   },
-  [types.SET_ENDPOINT_INFO](state: State, data: any) {
+  [types.SET_ENDPOINT_INFO](state: State, data: any = {}) {
```
Comment (TinyAllen) [inline src/store/modules/dashboard-endpoint.ts]: [types.SET_ENDPOINT_INFO](state: State, data: any) {
    if (!data) { data = {}; }

**Comment by TinyAllen** [comment]
LGTM

**Pull request merged**